### PR TITLE
Learning git commands.

### PR DESCRIPTION
**Pull request by sainipankaj90k**
Learning GIT.

### CODE REVIEW

File: js_july_23/Tutorial/0 - Git/Git_Commands.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// git add .
+// this is to add all in progress (checked out) files to commit.
+
+// git commit -m "Git download and repo structure"
+// this is to commit (burn) the changes in the branch.
+
+// git push
+// it pushes the changes to git-repo. 
+
+// git log
```
Comment (sainipankaj90k) [inline js_july_23/Tutorial/0 - Git/Git_Commands.txt:39]: Didn't understand what this command does.

File: js_july_23/Users/Sandeep/ReadMe.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Home-work.
+// Leet-code.
```
Comment (sandeepsainihisar) [inline js_july_23/Users/Sandeep/ReadMe.txt:2]: this is my commit pull request

File: js_july_23/Users/Manita/ReadMe.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Home-work.
```
Comment (manita1993) [inline js_july_23/Users/Manita/ReadMe.txt:1]: My First comment

File: js_july_23/Tutorial/0 - Git/Git_Commands.txt [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+// to see the commit history on the branch. 
+
+//// Folder structure : 'js_july_23' -> 'Tutorial' (contains tutorial content)
+//// Folder structure : 'js_july_23' -> 'Users/<nameOfuser> (contains content for the user)
```
Comment (sainipankaj90k) [inline js_july_23/Tutorial/0 - Git/Git_Commands.txt:43]: Add context about Pull Request too.

**Review by sainipankaj90k — COMMENTED** [review]
Added PR section as well.

**Review by sandeepsainihisar — APPROVED** [review]

File: js_july_23/Tutorial/0 - Git/Git_Commands.txt [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+// * main
+
+// How to create a new branch. <say, how to checkhout a new branch>.
+
+// git checkout -b users/pankaj/js_july_23/gitbasic
+// Format of branch is 'users/<name_of_user>/js_july_23/<name_of_your_branch>'
+
+// Now change the content as per need.
+
+// git status
```
Comment (Yuvi363uv) [inline js_july_23/Tutorial/0 - Git/Git_Commands.txt:27]: this is my 1st comment.

**Review by Yuvi363uv — APPROVED** [review]
hy this is my approve request

**Review by manita1993 — APPROVED** [review]
Approval from my side

**Pull request merged**